htdocs/core/lib/functions.lib.php

```diff
@@ -1922,9 +1922,23 @@ function dol_print_phone($phone,$countrycode='',$cid=0,$socid=0,$addlink='',$sep
 		$titlealt=($withpicto=='fax'?$langs->trans("Fax"):$langs->trans("Phone"));
 	}
 	$rep='';
+	$picto = '';
+	if($withpicto){
+		if($withpicto=='fax'){//should we use switch case instead ?
+			$picto = 'phoning_fax';
+		}elseif($withpicto=='phone'){
```

Comment:
hi
I don't want to create a new one for stantard phone (in this case i use the object_phoning.png picture. I've just want to add the  object_phoning_mobile.png `$picto = 'phoning_mobile'`.